tools/maf_clean_up.py

```diff
@@ -0,0 +1,177 @@
+input_maf = open ("data_mutations_extended.txt", "r")
```

Comment:
Another suggestion but not necessary - maybe we can let the users pass in files for input and output?

Reply:
fixed.